srcpkgs/vivaldi/template

```diff
@@ -18,9 +18,10 @@ nostrip=yes
 
 if [ "$XBPS_TARGET_MACHINE" = "x86_64" ]; then
 	_debarch=amd64
-	checksum=cd6648b4a6b45069e7bc4955cc0464abca6bd9c5c70f7a63f3a162e9e002f80c
+	checksum=7033110735bbd3cb69ce3b6d7daf42fd265b20c5dcc7a8f90f3f53569eadd38f
 else
 	_debarch=i386
+        version=3.8.2259.42
```

Comment:
Is there any official statement about vivaldi 4 not being available for i386? Or just not available temporary?

Reply:
Well, I couldn't download it. On the official download page, it's only vivaldi 3.7 (even if 3.8 is working).

I don't really know and I couldn't find anything on the forum. Maybe we should ask them directly ?

Reply:
An official statement would be nice, since having 2 version in a single template is ugly (and afaik breaks some tooling like xbps-checkvers, which isn't that extremely problematic, since this package is restricted)

Reply:
I created a thread on the [forum](https://forum.vivaldi.net/topic/62973/vivaldi-4-0-for-i386)

Reply:
So, I got an answer  : https://forum.vivaldi.net/topic/62973/vivaldi-4-0-for-i386/2#

_"Yes, but currently the 32bit versions for Vivaldi's Chromium core can not be built as this happens on some other Linux distributions, too."_